partiql-parser/build.gradle.kts

```diff
@@ -15,12 +15,13 @@
 plugins {
     id(Plugins.antlr)
     id(Plugins.conventions)
-    // id(Plugins.publish)
+    id(Plugins.publish)
 }
 
 dependencies {
     antlr(Deps.antlr)
     implementation(project(":partiql-ast"))
```

Comment:
Do you think `partiql-ast` should be `api`?

The reason for that is if someone consumes the `Parser` interface, the `parseAstStatement` method returns an API defined in `partiql-ast`.

Reply:
Yes. Good callout.